pom.xml

```diff
@@ -2111,7 +2258,7 @@ org.hibernate.search.spi.SearchIntegratorBuilder#buildSearchIntegrator()
         <profile>
             <id>elasticsearch-5.0</id>
             <properties>
-                <test.elasticsearch.mavenPlugin.version>5.7</test.elasticsearch.mavenPlugin.version>
+                <version.elasticsearch.plugin>5.7</version.elasticsearch.plugin>
                 <test.elasticsearch.host.version>5.1.2</test.elasticsearch.host.version>
                 <test.elasticsearch.host.url>http://localhost:9200</test.elasticsearch.host.url>
```

Comment:
I know it's unrelated to you changes, but could you remove this line while you're at it? It's useless, the value we set here is the default set higher in the same POM.

Reply:
Done.